src/Resources/Files.php

```diff
@@ -8,34 +8,45 @@ class Files extends Resource
      * Upload a new file.
      *
      * @param resource|string $file
-     * @param array           $params Optional parameters
+     *
+     * @see https://legacydocs.hubspot.com/docs/methods/files/v3/upload_new_file
      *
      * @return \SevenShores\Hubspot\Http\Response
      */
-    public function upload($file, array $params = [])
-    {
-        $endpoint = 'https://api.hubapi.com/filemanager/api/v2/files';
-
-        $queryString = build_query_string([
-            'overwrite' => isset($params['overwrite']) ? $params['overwrite'] : false,
-            'hidden' => isset($params['hidden']) ? $params['hidden'] : false,
-        ]);
-
-        $options['multipart'] = [
+    public function upload(
```

Comment:
Would you mind adding new params to method phpdoc signature?

Reply:
It was removed by cs fix